README.md

```diff
@@ -712,9 +720,12 @@ Example output:
     "created": [
         {
             "type": "file",
-            "metadataTxId": "YfdDXUyerPCpBbGTm_gv_x5hR3tu5fnz8bM-jPL__JE",
-            "dataTxId": "l4iNWyBapfAIj7OU-nB8z9XrBhawyqzs5O9qhk-3EnI",
+            "entityName": "file.txt"
             "entityId": "6613395a-cf19-4420-846a-f88b7b765c05"
+            "dataTxId": "l4iNWyBapfAIj7OU-nB8z9XrBhawyqzs5O9qhk-3EnI",
+            "metadataTxId": "YfdDXUyerPCpBbGTm_gv_x5hR3tu5fnz8bM-jPL__JE",
+            "bundledIn": "1zwdfZAIV8E26YjBs2ZQ4xjjP_1ewalvRgD_GyYw7f8-jPL__JE",
+            "sourceUri": "file://Users/BestArDriver/path/to/file.txt"
```

Comment:
Nit: given the input file which appears to start at the root of the file system, the sourceUri maybe ought to be `file://path/to/file.txt`.

Reply:
Included it here: f4e4bf3

I added it as `file:///path/to/file.txt` because my macOS system seems to only navigate when the path uses `file:///Users/...` . I suspect windows would be `file://C:/` or something